sky/cli.py

```diff
@@ -3041,6 +3048,8 @@ def _output():
             else:
                 yield ('\n\nHint: use -a/--all to see all accelerators '
                        '(including non-common ones) and pricing.')
+                if cloud_is_kubernetes and kubernetes_autoscaling:
+                    yield kubernetes_utils.KUBERNETES_AUTOSCALER_NOTE
```

Comment:
Does this mean when the `cloud` is not specified, i.e. None, we will not print the hint? Should we print the hint if `cloud is None` as well.

Also, in the show_all case, should we print this hint as well?

Reply:
ahh good point - fixed now! shown when cloud is not specified and also in show_all case.